internal/controller/operatorconfigmap_controller.go

```diff
@@ -185,6 +187,18 @@ func (c *OperatorConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Watches(&admrv1.ValidatingWebhookConfiguration{}, enqueueConfigMapRequest, webhookPredicates).
 		Watches(&v1alpha1.StorageClient{}, enqueueConfigMapRequest, builder.WithPredicates(predicate.AnnotationChangedPredicate{}))
 
+	if c.AvailableCrds[noobaaCrdName] {
```

Comment:
@leelavg I am not sure if all CRDs are loaded into AvailableCrds, or just the ones we are interested in. Depending on the answer, this code might or might not have a bug in it.